projects/baby_driver/scripts/test_can_send.py

```diff
@@ -0,0 +1,37 @@
+"""This Module Tests methods in can_send.py"""
+# pylint: disable=unused-import
+
+import unittest
+from unittest.mock import patch
+
+import can
+import can_util
+import cantools
+from can_send import can_send_raw, load_dbc, can_send
+
+
+class TestCanSendRaw(unittest.TestCase):
```

Comment:
Ideas for testing your functions:
* mock `can_util.send_message`, call `can_send_raw`, and just check that the parameters passed to `send_message` are what you expect
* mock `encode`, `get_message_by_name`, and `can_util.send_message`, call `can_send`, and check that `send_message` gets the parameters you expect - that'll check that all the data flow is working and avoid errors like the `msg_obj` thing above